docs/en/integrations/data-visualization/grafana/query-builder.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_label: Query Builder
+sidebar_position: 2
+slug: /en/integrations/grafana/query-builder
+description: Using the Query Builder in the ClickHouse Grafana plugin
+---
+
+# Query Builder
+
+Any query can be run with the ClickHouse plugin.
+The query builder is a convenient option for simpler queries, but for complicated queries you will need to use the [SQL Editor](#sql-editor).
+
+Queries in the query builder all have a [query type](#query-types), and require at least one column to be selected.
+
+## Query Types
+
+The *Query Type* setting will change the layout of the query builder to match the type of query being built.
+In explore view, the query type also determines which panel is used when visualizing data.
+
+### Table
+
+The most flexible query type is the table query. This is a catch-all for the other query builders designed to handle simple and aggregate queries.
+
+| Field | Description |
+|----|----|
+| Builder Mode  | Simple queries exclude Aggregates and Group By, while aggregate queries include these options.  |
+| Columns | The selected columns. Raw SQL can be typed into this field to allow for functions and column aliasing. |
+| Aggregates | A list of aggregate functions. Allows for custom values for function and column. Only visible in Aggregate mode. |
+| Group By | A list of `GROUP BY` names. Only visible in Aggregate mode. |
+| Order By | A list of `ORDER BY` names. |
```

Comment:
I'm wondering if we should link to the clause docs https://clickhouse.com/docs/en/sql-reference/statements/select/order-by here and in other places

Reply:
Good idea, there's probably other places where we can link to statements/functions too. I'll keep this in mind